apps/idos-enclave/src/pages/App.tsx

```diff
@@ -64,6 +65,8 @@ export function App({ store, enclave }: AppProps) {
   const messageReceiver = useCallback((event: MessageEvent<EventData>) => {
     if (event.source !== enclave) return;
 
+    setCurrentUserPublicKey(event.data.message.currentUserPublicKey);
```

Comment:
I think we should move this to be under `case "auth"`, since that's the only message that gets this value.